README.md

```diff
@@ -1,2 +1,131 @@
 # analytics-ios-integrations
-Monorepo storing Segment's analytics iOS integrations
+
+Monorepo storing Segment's analytics iOS integrations. You can find additional documentation for this repo in the `guides` directory.
+
+## Requirements:
+To run monorepo or migrate a destination, the following are required:
+
+Xcode 8+
+Xcode command line tools and default must be set under "Locations" in Xcode preferences. (If multiple versions of Xcode are installed on a machine, an occur may occur, and so correct default version must be correctly selected)
+
+And local dependencies:
+
+1- HomeBrew
+2- Cocopods 1.8
+3- RubyGems Bundler
+4- OCLint
+5- Clang-format
+
+Alternatively, run .check.sh to install any missing local dependencies.
+
+```bash
+$ ./check.sh
+```
+
+## Usage
+You can open the iOS-Integrations.xcworkspace project on Xcode. All integrations are included by default.
+
+Any scripts can be either run in "Scripts" directory or in root directory.
+
+To run in root, use the following:
+
+```bash
+$ ./start.sh sh xxxx.sh
+```
+For the purposes of the guide, we will assume, you are inside the Scripts directory.
+
+1- Please run check.sh to make sure all required dependencies have been installed on machine
```

Comment:
@briemcnally I added in Usage how to run within Scripts directory or in root of project.